x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/reducer.ts

```diff
@@ -291,6 +284,19 @@ export const endpointListReducer: StateReducer = (state = initialEndpointPageSta
     const wasPreviouslyOnListPage = isOnEndpointPage(state) && !hasSelectedEndpoint(state);
     const isCurrentlyOnDetailsPage = isOnEndpointPage(newState) && hasSelectedEndpoint(newState);
     const wasPreviouslyOnDetailsPage = isOnEndpointPage(state) && hasSelectedEndpoint(state);
+    const wasPreviouslyOnActivityLogPage =
```

Comment:
wasn't this moved into a selector?

Reply:
no not the whole thing. I kept the same pattern as the other vars. I made a new `getIsOnEndpointDetailsActivityLog ` selector that compares the `show` param to tab id. [here](https://github.com/elastic/kibana/pull/105716/files/1dc39658ec5133e761065f1deb716a8a4e0afad3#diff-279b87c8ce4f114b9f10ae5f909e5f1853b0b70c208ca0420163c38f0bd72bc5R373)